x/evm/statedb/statedb.go

```diff
@@ -382,6 +386,22 @@ func (s *StateDB) Context() sdk.Context {
  * SETTERS
  */
 
+// Transfer from one account to another
+func (s *StateDB) Transfer(sender, recipient common.Address, amount *big.Int) {
+	if amount.Sign() == 0 {
```

Comment:
should we align with [other](https://github.com/crypto-org-chain/ethermint/blob/a76e99d87ddbc9ff75e132f7d0b8efc92b820072/x/evm/statedb/statedb.go#L407) check

Reply:
maybe we should do a panic if it's negative

Reply:
done